### PR TITLE
Replaced input placeholder's with e.g.

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at providing an e.g. as inputs placeholder.
I did not do the create/update password inputs, as it will be done by Pauline during this sprint.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

### UserProfile

image

image

---

### UserAddress

image

image

---

### Email Identity

image

image

image

image

---

### Phone Identity

image

image

image

image

## Checklist

- [ ] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-9y0m3x [chore] - Improve input labels/placeholders](https://app.clickup.com/t/9y0m3x)

**Review by Meyclem — APPROVED** [review]

File: content/locales/en.ts [hunk 1]
```diff
@@ -36,6 +36,13 @@ const addressInputStrings = {
   cancel: "Cancel",
 };
 
+const identitiesInputStrings = {
+  emailPlaceholder: "frieda@ewlines.tech",
+  phonePlaceholder: "01632 960049",
```
Comment (Oluane) [inline content/locales/en.ts:41]: Is it a random fake value ? Or could it be a real phone number somewhere ?

Comment (Fuzznimp) [reply content/locales/en.ts:41]: This is a fake one 🙂.

Comment (Fuzznimp) [reply content/locales/en.ts:41]: The English address is also a fake one.

**Pull request merged**